.circleci/config.yml

```diff
@@ -0,0 +1,66 @@
+# Note: all paths are relative to the default workspace ~/project
+version: 2.1
+executors:
+  node-11:
```

Comment:
can we rename to node-10, since that's what it is?

Reply:
Set executor to node v11

Reply:
Please use stable releases, 10 is good.